src/common/stats/test/StatsManagerCrossLevelTest.cpp

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include <gtest/gtest.h>
+#include "stats/StatsManager.h"
+#include "thread/GenericWorker.h"
+
+namespace nebula {
+namespace stats {
+
+TEST(StatsManager, CrossLevelTest) {
+    auto statId = StatsManager::registerHisto("stat03", 1, 1, 100);
+    std::vector<std::thread> threads;
+    for (int i = 0; i < 10; i++) {
+        threads.emplace_back([statId, i] () {
+            for (int k = i * 10 + 10; k >= i * 10 + 1; k--) {
+                StatsManager::addValue(statId, k);
+                if (k > i * 10 + 1) {
+                    sleep(7);
```

Comment:
why sleep 7s?  this test will cost more than 1 min

Reply:
Yes, it's costly. But the durations are predefined, i.e. 60, 600, etc. Here for the following assertions, we are testing on the minimal duration of 60s.